### PR TITLE
Reformatted and reworked stdlib documentation. Added builtins

### PR DESCRIPTION
**Pull request by radrow**
I added all builtin functions I know to the sophia stdlib doc. Also I reformatted it to make it display more nicely and make it easier to automatically reprocess by some script. Also fixed some overlooks.

### CODE REVIEW

File: contracts/sophia_stdlib.md [hunk 1]
```diff
@@ -1,95 +1,711 @@
...
+
+
+## Address
+
+### to_str
+```
+Address.to_str(a : address) : string
+```
+
+Base58 encoded string
```
Comment (hanssv) [inline contracts/sophia_stdlib.md]:
```suggestion
Base58Check encoded string
```

File: contracts/sophia_stdlib.md [hunk 2]
```diff
@@ -1,95 +1,711 @@
...
+signing the oracle query id + contract address
+
+
+### extend
+```
+Oracle.extend(<signature : bytes(64)>, o : oracle('a, 'b), ttl : Chain.ttl) : unit
+```
+
+Extends TTL of an oracle.
+* `singature` is a named argument and thus optional. Must be the same as for `Oracle.register`
```
Comment (hanssv) [inline contracts/sophia_stdlib.md]:
```suggestion
* `signature` is a named argument and thus optional. Must be the same as for `Oracle.register`
```

Comment (radrow) [reply contracts/sophia_stdlib.md]: Good catch!

**Comment by tolbrino** [comment]
@radrow Isn't this PR replaced by https://github.com/aeternity/aesophia/pull/239 ?

**Comment by radrow** [comment]
Yes, it is

**Pull request closed without merging**